plugin-desktopswitch/desktopswitchbutton.cpp

```diff
@@ -51,6 +51,10 @@ void DesktopSwitchButton::update(int index, LabelType labelType, const QString &
             setText(title);
             break;
 
+        case LABEL_TYPE_NONE:
+            setText(" ");
```

Comment:
It is all done. I re-uploaded the patch and rebased to current lxqt-panel master.